vllm_hpu_extension/bucketing.py

```diff
@@ -221,6 +221,12 @@ def generate_prompt_buckets(bs_bucket_config,
             return list(
                 sorted(buckets, key=lambda b: (b[0] * b[1], b[1], b[0]))), []
 
+    # We remove sequences we do not want to batch: via VLLM_MAX_BATCHED_PROMPT_LEN 
+    # i.e eliminate prompt buckets with bs > 1 and seqlen > VLLM_MAX_BATCHED_PROMPT_LEN
+    import sys
+    max_batched_seq_len = int(os.environ.get('VLLM_MAX_BATCHED_PROMPT_LEN', sys.maxsize))
+    filtered_buckets = list(filter(lambda bucket : bucket[0]==1 or bucket[1] <= max_batched_seq_len, filtered_buckets))
```

Comment:
@attafosu , can you also add another env var for Batch_size, limit to 1 is not working for 70B models.